### ISSUE TITLE
Error download account name, details logged to DebugInfo.log

### ISSUE THREAD

**Issue by Kinroko** [Stickymaddness/Procurement#941]
[Error] Error downloading account name, details logged to DebugInfo.log. Please open a ticket at https://github.com/Stickymaddness/Procurement/issues and include your DebugInfo.log
[DebugInfo.log] 

(https://github.com/Stickymaddness/Procurement/files/2807305/DebugInfo.log)

**Comment by aydjay** [comment]
You look to be getting 503 (Server Unavailable)

[29-01-2019 5:49] Failed to build HTTP request and get response for: method=GET, url='https://www.pathofexile.com/login', allowAutoRedirects=, requestData='': The remote server returned an error: (503) Server Unavailable.

Try again tomorrow.

**Comment by Kinroko** [comment]
Is there any chance this could be caused by trying to update the shop too frequently/quickly? Or possibly one after the other? I had just updated my shop when I tried to bump, and received a similar error about my DebugInfo.log but I was still able to access the functions of Procurement. 

Once I tried to update my stash tabs fully it entered an infinite loading loop and on restart I got the error message you saw in my logs.
